HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -3472,8 +3471,7 @@ def is_exterior
     def is_exposed
       if HPXML::is_conditioned(self) &&
          (@exterior_adjacent_to == LocationOutside ||
-          @exterior_adjacent_to == LocationOtherNonFreezingSpace ||
-          @exterior_adjacent_to == LocationGarage)
+          @exterior_adjacent_to == LocationOtherNonFreezingSpace)
         return true
```

Comment:
It looks like there is similar logic in three places, which is a recipe for introducing bugs in the future. Can you create a single method (similar to e.g. `HPXML::is_conditioned`)?